### PR TITLE
fix: deniability button now disables the comments field

### PR DESCRIPTION
**Pull request by Retengart**
Now the comments field will be inactive when the "deniability" button is pressed. I also showed the binary code of the encrypted file to confirm that "deniability" is working

https://github.com/user-attachments/assets/30710a5d-4125-4f5d-8ac9-5212f3ae075b

### CODE REVIEW

**Comment by HACKERALERT** [comment]
This is great, thanks! For version though, please make it 1.39, otherwise the regex matcher won't detect volumes as Picocrypt volumes.

File: src/Picocrypt.go [hunk 1]
```diff
@@ -515,7 +517,7 @@ func draw() {
 
 					giu.Row(
 						giu.Checkbox("Deniability", &deniability),
-						giu.Tooltip("Add plausible deniability to the volume"),
+						giu.Tooltip("Add plausible deniability to the volume\nIf enabled, the comments will be lost"),
```
Comment (HACKERALERT) [inline src/Picocrypt.go:520]: Some people may not use comments..so maybe better to say

\nIf enabled, comments will not work.

Or something like that

**Comment by Retengart** [comment]
> This is great, thanks! For version though, please make it 1.39, otherwise the regex matcher won't detect volumes as Picocrypt volumes. (Try decrypting a volume when it is made with a 1.38.1 version)

The version was changed only in your copyright header and VERSION file, I didn't touch "version" variable, so volumes have standard 1.38 version in binary header.
image

**Comment by HACKERALERT** [comment]
Missed that. Still a good idea to stay consistent and keep code and version tag in sync though. No shortage of version numbers to use, so use them :)

**Comment by Retengart** [comment]
All is done, I used "git commit --amend"

**Review by HACKERALERT — APPROVED** [review]

**Comment by HACKERALERT** [comment]
Awesome. Ready to merge?

**Comment by Retengart** [comment]
Yeah, I think so

**Pull request merged**